### PR TITLE
Support for custom images.

### PR DESCRIPTION
**Pull request by treeder**

### CODE REVIEW

**Comment by treeder** [comment]
Requires: https://github.com/iron-io/iron_go/pull/47

**Comment by rdallman** [comment]
seems not working as you have it laid out `iron worker upload [--zip [ZIPFILE]] [IMAGE] [COMMAND]`, e.g.: 

```shell
ironcli(custom-images) ✗: ./ironcli --env prod worker upload treeder/magic        
file extension must be .zip, got: treeder/magic
```

**Comment by rdallman** [comment]
need to add `-zip` to `flags.go`

File: commands.go [hunk 1]
```diff
@@ -407,33 +408,57 @@ func (u *UploadCmd) Flags(args ...string) error {
...
+		u.cmd = &c
+	} else {
+		im := u.flags.Arg(0)
+		u.image = &im
+		u.stack = nil
+		// command is optional
+		if u.flags.NArg() > 1 {
+			// TODO move "command" field into "worker.Code"
+			c := strings.Join(u.flags.Args()[1:], " ")
+			u.cmd = &c
```
Comment (rdallman) [inline commands.go]: assign to `u.codes.Command` instead?

**Comment by rdallman** [comment]
@treeder https://github.com/iron-io/ironcli/commit/83e350f7c54f5b61d19710221b019f5f9518487b should fix everything

**Comment by rdallman** [comment]
we need a swapi to test on

**Comment by rkononov** [comment]
@rdallman swapi deployed to staging (staging-hud.iron.io and swapi host - staging-worker.iron.io)
we deployed most recent version from https://github.com/iron-io/go/pull/95
also if you want to deploy anything else - just merge pr into master of go repo and deploy on these servers - http://www.simpledeployer.com/projects/77a25c4d-463f-441d-b4cf-25052bf3a899/servers

**Comment by rdallman** [comment]
thanks @rkononov

**Pull request merged**